### PR TITLE
[bn] Bengali Locale Added

### PR DESCRIPTION
**Pull request by hafijul233**
I have added the bengali language translation for all of the message. Here is the list
* Cache
* Cast
* CLI
* Cookie
* Core
* Database
* Email
* Encryption
* Fabricator
* Files
* Filters
* Format
* HTTP
* Images
* Log
* Migrations
* Number
* Pager
* Publisher
* RESTful
* Router
* Security
* Session
* Test
* Time
* Validation
* View

### CODE REVIEW

File: tests/Language/BengaliTranslationTest.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace Language;
```
Comment (kenjis) [inline tests/Language/BengaliTranslationTest.php:12]:
```suggestion
namespace Translations\Tests;
```

File: tests/Language/AbstractTranslationTestCase.php [hunk 2]
```diff
@@ -12,6 +12,7 @@
 namespace Translations\Tests;
 
 use CodeIgniter\CLI\CLI;
+use Language\BengaliTranslationTest;
```
Comment (kenjis) [inline tests/Language/AbstractTranslationTestCase.php:15]: Please remove this line.

File: tests/Language/BengaliTranslationTest.php (hunk 1 above)
Comment (hafijul233) [reply tests/Language/BengaliTranslationTest.php:12]: patched and pushed

File: tests/Language/AbstractTranslationTestCase.php (hunk 2 above)
Comment (hafijul233) [reply tests/Language/AbstractTranslationTestCase.php]: Removed this line also

File: README.md [hunk 3]
```diff
@@ -38,6 +38,7 @@ claims no expertise beyond English.
 |--------|---------------------|---------|-----------------------| --------------------
 | ar     | Arabic              | 4.0.0   | [daif][ar]            | Daif Alotaibi
 | bg     | Bulgarian           | 4.2.2   | [ivcho02][bg]         | Ivaylo Tsvetanov
+| bn     | Bengali             | 4.2.11  | [hafijul233][bn]      | Hafijul Islam
```
Comment (paulbalandan) [inline README.md:41]: change the version to 4.3.0

Comment (hafijul233) [reply README.md:41]: Okay.

**Comment by kenjis** [comment]
```
1) Translations\Tests\AutoReview\TranslationsCodeTest::testLocalesArrayAreArrangedByLocaleCode
The `AbstractTranslationTestCase::$locales` property should be arranged by locale code.
Failed asserting that two arrays are identical.
--- Expected
+++ Actual
@@ @@
 Array &0 (
     'Translations\Tests\ArabicTranslationTest' => 'ar'
+    'Translations\Tests\BengaliTranslationTest' => 'bn'
     'Translations\Tests\BulgarianTranslationTest' => 'bg'
-    'Translations\Tests\BengaliTranslationTest' => 'bn'
     'Translations\Tests\BosnianTranslationTest' => 'bs'
     'Translations\Tests\CzechTranslationTest' => 'cs'
     'Translations\Tests\GermanTranslationTest' => 'de'

/home/runner/work/translations/translations/tests/AutoReview/TranslationsCodeTest.php:34
phpvfscomposer:///home/runner/work/translations/translations/vendor/phpunit/phpunit/phpunit:97
```
https://github.com/codeigniter4/translations/actions/runs/3918893688/jobs/6700568666

**Comment by kenjis** [comment]
```
1) Translations\Tests\BengaliTranslationTest::testAllConfiguredLanguageFilesAreTranslated with data set "bn" ('bn')
Failed asserting that language file "Errors.php" in the main repository is translated in "bn" locale.
Failed asserting that an array is empty.

/home/runner/work/translations/translations/tests/Language/AbstractTranslationTestCase.php:109
phpvfscomposer:///home/runner/work/translations/translations/vendor/phpunit/phpunit/phpunit:97

2) Translations\Tests\BengaliTranslationTest::testAllIncludedLanguageKeysAreTranslated with data set "bn" ('bn')
Failed asserting that the translated language key "Number.quadrillion" in "bn" locale differs from the original keys in the main repository.
Failed asserting that an array is empty.

/home/runner/work/translations/translations/tests/Language/AbstractTranslationTestCase.php:257
phpvfscomposer:///home/runner/work/translations/translations/vendor/phpunit/phpunit/phpunit:97
```
https://github.com/codeigniter4/translations/actions/runs/3918893697/jobs/6700550315

**Comment by kenjis** [comment]
@hafijul233 Can you fix for the failed checks?

**Comment by kenjis** [comment]
@hafijul233 You forgot GPG signing. Please sign all your commits.
See 
- https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#signing
- https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#gpg-signing-old-commits

**Comment by hafijul233** [comment]
> @hafijul233 Can you fix for the failed checks?

I have fixed it. https://github.com/hafijul233/ci4-translations/commit/43061e769894955eb6fd58c1671d9d86102bfb8b

**Comment by hafijul233** [comment]
> ```
> 1) Translations\Tests\AutoReview\TranslationsCodeTest::testLocalesArrayAreArrangedByLocaleCode
> The `AbstractTranslationTestCase::$locales` property should be arranged by locale code.
> Failed asserting that two arrays are identical.
> --- Expected
> +++ Actual
> @@ @@
>  Array &0 (
>      'Translations\Tests\ArabicTranslationTest' => 'ar'
> +    'Translations\Tests\BengaliTranslationTest' => 'bn'
>      'Translations\Tests\BulgarianTranslationTest' => 'bg'
> -    'Translations\Tests\BengaliTranslationTest' => 'bn'
>      'Translations\Tests\BosnianTranslationTest' => 'bs'
>      'Translations\Tests\CzechTranslationTest' => 'cs'
>      'Translations\Tests\GermanTranslationTest' => 'de'
> 
> /home/runner/work/translations/translations/tests/AutoReview/TranslationsCodeTest.php:34
> phpvfscomposer:///home/runner/work/translations/translations/vendor/phpunit/phpunit/phpunit:97
> ```
> 
> https://github.com/codeigniter4/translations/actions/runs/3918893688/jobs/6700568666

I have fixed it also

**Pull request closed without merging**